pkg/channel/consolidated/utils/util.go

```diff
@@ -62,24 +63,7 @@ type KafkaConfig struct {
 	AuthSecretNamespace string
 }
 
-type KafkaAuthConfig struct {
-	TLS  *KafkaTlsConfig
-	SASL *KafkaSaslConfig
-}
-
-type KafkaTlsConfig struct {
-	Cacert   string
-	Usercert string
-	Userkey  string
-}
-
-type KafkaSaslConfig struct {
-	User     string
-	Password string
-	SaslType string
-}
-
```

Comment:
Moved these into `common` to be able to reuse them in 2 channels and the source.